pygmo/docstrings.cpp

```diff
@@ -629,6 +659,83 @@ of a :class:`~pygmo.core.problem`.
 )";
 }
 
+std::string problem_get_nc_docstring()
+{
+    return R"(get_nc()
+
+Total number of constraints.
+
+This method will return the sum of the output of :func:`~pygmo.core.problem.get_nic()` and
+:func:`~pygmo.core.problem.get_nec()` (i.e., the total number of constraints).
+
+Returns:
+    ``int``: the total number of constraints of the problem
+
+)";
+}
+
+std::string problem_c_tol_docstring()
+{
+    return R"(Constraints tolerance.
+
+This property contains an array of ``float`` that are used when checking for constraint feasibility.
+The dimension of the array is :math:`n_{ec} + n_{ic}`, and the array is zero-filled on problem
+construction.
+
+Returns:
+    1D NumPy float array: the constraints tolerance
```

Comment:
We have to agree on some format for these numpy types. Elsewhere I used confusing types. I guess we take this problem review as baseline and change the rest?

array-like -> when its an arg taking any iterable
xD NumPy float array, xD NumPy int array in other cases

in all cases for these types we do not use the quotes?